Add new Benchmark home pages (#9635)
opensearch-project/documentation-website: _benchmark/user-guide/optimizing-benchmarks/index.md

nav_order: 25
parent: User guide
has_children: true
more_cards: 
  - heading: "Running distributed loads"
    description: "Configure multiple load generator machines to run large-scale benchmarks"
    link: "/benchmark/user-guide/optimizing-benchmarks/distributed-load/"
  - heading: "Expanding a workload's data corpus"
    description: "Increase dataset size and complexity for more realistic benchmark scenarios"
    link: "/benchmark/user-guide/optimizing-benchmarks/expand-data-corpus/"
  - heading: "Target throughput"
    description: "Set specific throughput targets to match your production workload patterns"
    link: "/benchmark/user-guide/optimizing-benchmarks/target-throughput/"
  - heading: "Performance testing best practices"

>     description: "Follow these principals to achieve more meaningful benchmark results"

Reviewer: ```suggestion
    description: "Follow these best practices to achieve more meaningful benchmark results"
```

    link: "/benchmark/user-guide/optimizing-benchmarks/performance-testing-best-practices/"
  - heading: "Randomizing queries"
    description: "Generate varied search patterns to simulate real-world queries"
    link: "/benchmark/user-guide/optimizing-benchmarks/randomizing-queries/"
---

# Optimizing benchmarks

> This section details different ways you can optimize the benchmark tools for your cluster.

Reviewer: ```suggestion
This section provides information about optimizing the benchmark tools for your cluster.
```
